lib/portage/eapi.py

```diff
@@ -353,9 +368,7 @@ def _decorate_eapi_funcs():
 _decorate_eapi_funcs()
 
 
-_eapi_attrs_cache = {}
-
-
+@cache
```

Comment:
You've seemed to mix changes from the other commit here.

Reply:
Great catch! I'll fix this.

Reply:
Ok this should be fixed now.